### DISCUSSION TITLE
Fitting of 3 filter dataset

### DISCUSSION THREAD

**Discussion by MohammadReza0GH** [phoebe-project/phoebe2#885] [Questions]
hi, i have 3 dataset in SDSS(g,i,r) and i want to to fit at the same time and wanna just one result for 3 filters, how can i do this?

**Comment by aprsa** [comment]
That is the default behavior in phoebe.

**Reply by MohammadReza0GH** [reply]
so i just need to add 3 dataset?

**Reply by aprsa** [reply]
Precisely.
